src/portal/src/i18n/lang/en-us-lang.json

```diff
@@ -1028,6 +1029,41 @@
         "IN_PROGRESS": "Search...",
         "BACK": "Back"
     },
+    "SBOM": {
+        "CHART": {
+            "SCANNING_TIME": "Scan completed time:",
```

Comment:
This is not clear: It is the time it took to complete the scan or is it the time when the scan terminated?